### ISSUE TITLE
Bookmark expiry timer.

### ISSUE THREAD

**Issue by piyusharmap** [gokulkrishh/bmrk.cc#5]
**Is your feature request related to a problem? Please describe.**
There are times when we bookmark something but forget to remove it after its usage. This clutters up our bookmark records.

**Describe the solution you'd like**
We can set a specific date range for a bookmark, and after that, we can remove it from the main list and place it in an expired list for, let's say, 15 days. After that period, we can permanently delete it from the database.

**Comment by gokulkrishh** [comment]
May i know how useful this will be? This is first time i hearing a feature like this in any app.

**Comment by piyusharmap** [comment]
Yes, not many apps need this feature but as there is a limited number of bookmarks per month so maybe it's helpful for the user if they have a choice to set an expiry timer for a bookmark because unlike social media posts there are bookmarks which we require for a specific use and for a specific time.

Actually, I have a lot of bookmarks but not all of them are useful so I thought maybe there could a feature to automatically remove them after some time.

**Comment by gokulkrishh** [comment]
Oh i see, let me think over this and get back. I am not sure it will be a useful feature for everyone. For now i am focusing features like sharing bookmarks, duplicate detection, full text search etc,
